### ISSUE TITLE
Reserved keyword bind breaks importcpp pragma

### ISSUE THREAD

**Issue by zacharycarter** [nim-lang/Nim#8728]
```
zachcarter@A-TX39FHTD6  ~/projects/sell-my-car-backend   master ●  nim -version
Nim Compiler Version 0.18.1 [MacOSX: amd64]
Compiled at 2018-08-22
Copyright (c) 2006-2018 by Andreas Rumpf

active boot switches: -d:release
```

`import jsffi proc bindTo(p: proc(), self: JsObject, args: JsObject): proc() {. importcpp: "bind" .}`

results in - 

`Error: attempting to call undeclared routine: 'importcpp`

which is not very clear at all.

https://nim-lang.org/docs/jsffi.html#bindMethod.m,typed - exists but users for the JS target are going to either desire a clearer error message here so they know they need to work around this with meta programming, or some type of implementation of bindMethod that also passes arguments to the closure.

**Comment by LemonBoy** [comment]
Not quite, you just have to wrap with parentheses the return type `(proc())` otherwise the `importcpp` pragma is wrongfully attached to it rather than `bindTo`.

**Comment by metagn** [comment]
Now gives `invalid pragma: importcpp: "bind"`. Maybe make it say `invalid pragma for proc(): importcpp: "bind"`?

[Relevant else branch](https://github.com/nim-lang/Nim/blob/31ba27590c6a2ddd5e6b65247b2d9f74b8ca1ba2/compiler/pragmas.nim#L1184-L1191)
